ext/ffi/00_ffi.js

```diff
@@ -170,11 +177,35 @@ class UnsafePointerView {
 const OUT_BUFFER = new Uint32Array(2);
 const OUT_BUFFER_64 = new BigInt64Array(OUT_BUFFER.buffer);
 class UnsafePointer {
+  static create(value) {
+    return ops.op_ffi_ptr_create(value);
+  }
+
+  static equals(a, b) {
+    if (a === b) {
+      return true;
+    } else if (a === null || b === null) {
+      return a === b;
+    }
```

Comment:
a is null, b is not
- `a === b` always false

a is not null, b is null
- `a === b` always false

a is null, b is null
- `a === b` always true

so the first `if` branch should also just cover the null cases right?

Reply:
Good point! Will fix.